app-backup/luckybackup/luckybackup-0.4.9.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="A powerful, fast and reliable backup & sync tool"
+DESCRIPTION="An application that backs-up and/or synchronizes any directories with the power \
+of rsync. It is simple to use, fast (transfers over only changes made and not all data), safe\
+ (keeps your data safe by checking all declared directories before proceeding in any data \
+manipulation), reliable and fully customizable."
```

Comment:
Sorry, I should have checked with a text editor. This description needs to be wraped so that it won't breach 80-collumn limit. Below is a wrapped version for your convenience. 

```sh
DESCRIPTION="An application that backs-up and/or synchronizes any directories \
with the power of rsync. It is simple to use, fast (transfers over only \
changes made and not all data), safe (keeps your data safe by checking all \
declared directories before proceeding in any data manipulation), reliable and \
fully customizable."
```

Reply:
ok.. updated